qemu/tests/numa_stress.py

```diff
@@ -87,6 +87,8 @@ def run(test, params, env):
     most_used_node, memory_used = max_mem_map_node(host_numa_node, qemu_pid)
 
     for test_round in range(test_count):
+        if os.path.isfile(memory_file):
```

Comment:
Agree with CongLi, in my test: the value of "os.path.isfile(memory_file)" is "False" all the time.
I think this file is created in line 105:
 utils.system(numa_dd_cmd, timeout=dd_timeout)